tutorial.md

```diff
@@ -16,30 +29,29 @@ metadata:
   name: my-isp
   namespace: default
 spec:
-  imageWhitelist:
-  - gcr.io/kritis-int-test/nginx-digest-whitelist:latest
-  - gcr.io/kritis-int-test/nginx-digest-whitelist@sha256:56e0af16f4a9d2401d3f55bc8d214d519f070b5317512c87568603f315a8be72
   packageVulnerabilityRequirements:
-    maximumSeverity: HIGH
-    onlyFixesNotAvailable: true
+    maximumSeverity: MEDIUM
     whitelistCVEs:
-      - providers/goog-vulnz/notes/CVE-2017-1000082
       - providers/goog-vulnz/notes/CVE-2017-1000081
 EOF
 ```
 
-The `ImageSecurityPolicy` should be configured:
+### 2. Copy a vulnerable image
+
+The [Container Analysis API](https://cloud.google.com/container-analysis/api/reference/rest/) only reveals vulnerability information for images owned by your project. This makes a copy of a sample vulnerable image into your container registry:
+
 ```shell
-imagesecuritypolicy.kritis.grafeas.io/my-isp configured
+gcloud container images add-tag \
+  gcr.io/kritis-tutorial/java-with-vuln:latest \
+  gcr.io/$PROJECT/java-with-vuln:latest
 ```
-This `ImageSecurityPolicy` specifies two `nginx` images that are whitelisted and should always be allowed entry.
-It sets the maximum CVE severity allowed in any image to `HIGH`, and whitelists two CVEs which should be ignored during validation.
-It also sets `onlyFixesNotAvailable: true`, meaning that images that contain CVEs with fixes available should be denied.
 
-### 2. Deploying An Image With Vulnerabilities
-Now that we have kritis installed and an `ImageSecurityPolicy` to validate against, we can go ahead and start deploying some images!
+For more information about copying images, see [Google Cloud: Pushing and Pulling Images](https://cloud.google.com/container-registry/docs/pushing-and-pulling).
+
```

Comment:
I've noticed that sometimes the vulnerability scanning is complete within a few seconds and sometimes it takes a few minutes, maybe we could add a note to wait for scanning to finish before proceeding?

Reply:
Great idea! I've added a mention of needing to wait for the vulnerabilities to show up in gcloud after copying the image in. PTAL.